draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -309,6 +312,8 @@ the Key Binding JWT, the `cnf` claim of the SD-JWT MUST be used.
 Furthermore, the recipient of the SD-JWT VC MUST validate the public verification key
 for the Issuer-signed JWT as defined in (#issuer-signed-jwt-verification-key-validation).
 
+If Schema Type Metadata is provided, a recipient MUST validate the JSON Schema as defined in (#schema-type-metadata).
```

Comment:
"Schema" should only be uppercased when used in "JSON Schema", otherwise I'd use lowercase (unless we define the term).